### DISCUSSION TITLE
Callback function

### DISCUSSION THREAD

**Discussion by andrconstruction** [react-hook-form/react-hook-form#2526] [Q&A]
Sorry for a simple question but how to use a callback function within controller.
I want to change other fields depends on one field, for it I  want ot call my function (rusToLat), but it is not fired

    <Controller
                            as={<Form.Control isInvalid={errors.display_name} placeholder={"Иванов Иван Иванович"}/>}
                            name="display_name"
                            control={control}
                            onBlur={() => **rusToLat**()  }
                            defaultValue={user.display_name}
                        />

**Comment by bluebill1049** [comment]
you should `render` props

https://react-hook-form.com/api#Controller
